DependencyInjection/Security/Factory/OAuthFactory.php

```diff
@@ -46,33 +45,15 @@ public function createAuthenticator(ContainerBuilder $container, string $id, arr
     /**
      * {@inheritdoc}
      */
-    public function create(ContainerBuilder $container, $id, $config, $userProvider, $defaultEntryPoint)
+    public function getPriority(): int
     {
-        $providerId = 'security.authentication.provider.fos_oauth_server.'.$id;
-        $container
-            ->setDefinition($providerId, new ChildDefinition('fos_oauth_server.security.authentication.provider'))
-            ->replaceArgument(0, new Reference($userProvider))
-            ->replaceArgument(2, new Reference('security.user_checker.'.$id))
-        ;
-
-        $listenerId = 'security.authentication.listener.fos_oauth_server.'.$id;
-        $container->setDefinition($listenerId, new ChildDefinition('fos_oauth_server.security.authentication.listener'));
-
-        return [$providerId, $listenerId, 'fos_oauth_server.security.entry_point'];
-    }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function getPosition()
-    {
-        return 'pre_auth';
+        return 0;
```

Comment:
Class OAuthFactory" should implement method AuthenticatorFactoryInterface::getPriority(): int
Defines the position at which the authenticator is called.

Grabbed the function from vendor/symfony/security-bundle/DependencyInjection/Security/Factory/CustomAuthenticatorFactory.php